src/WebJobs.Script/Environment/EnvironmentExtensions.cs

```diff
@@ -471,10 +471,10 @@ public static string GetFunctionsWorkerRuntime(this IEnvironment environment)
         /// <summary>
         /// Gets a value indicating whether AzureFileShare should be mounted when specializing Linux Consumption workers.
         /// </summary>
-        public static bool SupportsAzureFileShareMount(this IEnvironment environment)
+        public static bool SupportsAzureFileShareMount(this IEnvironment environment, string runFromPackageValue)
         {
             return string.Equals(environment.GetFunctionsWorkerRuntime(), RpcWorkerConstants.PowerShellLanguageWorkerName,
-                StringComparison.OrdinalIgnoreCase);
+                StringComparison.OrdinalIgnoreCase) || string.Equals(runFromPackageValue, "1", StringComparison.OrdinalIgnoreCase);
```

Comment:
>|| string.Equals(runFromPackageValue, "1", StringComparison.OrdinalIgnoreCase) [](http://example.com/codeflow?start=52&length=78)

lets move this to its own method IsLocalRunFromPackage and there should be no need to pass runFromPackageValue and instead it should be available from environment

Reply:
let callers do SupportsAzureFileShareMount || IsLocalRunFromPackage

Reply:
I changed it recommended.